homeassistant/components/lock/wink.py

```diff
@@ -0,0 +1,68 @@
+"""
+homeassistant.components.lock.wink
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Wink locks.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/lock.wink/
+"""
+import logging
+
+from homeassistant.components.lock import LockDevice
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+REQUIREMENTS = ['https://github.com/balloob/python-wink/archive/'
```

Comment:
We should point this at the python-wink version with the lock support. We need to update this too for `wink.py`, `light/wink.py`, `switch/wink.py`, `sensor/wink.py`.
```python
REQUIREMENTS = ['https://github.com/balloob/python-wink/archive/'
                '42fdcfa721b1bc583688e3592d8427f4c13ba6d9.zip'
                '#python-wink==0.2']
```
```